seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -365,8 +392,8 @@
       "challengeType": 5,
       "nameEs": "Repite el texto Repite el texto",
       "descriptionEs": [
-        "Repite una cadena de texto dada (primer argumento) n veces (segundo argumento). Devuelve una cadena de texto vacía si n es un número negativo.",
-        "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
+        "Repite una cadena de texto dada (primer argumento) <code>num</code> veces (segundo argumento). Retorna una cadena de texto vacía si <code>num</code> es un número negativo.",
```

Comment:
All previous exercises use "devolver" for returning values. For consistency I would keep `devuelve `instead of 

> Retorna 

for this one.